pandas/tests/indexes/common.py

```diff
@@ -51,6 +51,21 @@ def test_to_series(self):
         assert s.index is not idx
         assert s.name == idx.name
 
+    def test_to_frame(self):
+        # see gh-15230
+        idx = self.create_index()
+        name = idx.name or 0
```

Comment:
There is at least one in the `create_index` indices that has a name? (or at least one that has no name)

Reply:
Yes, there is.